client/views/create/create.js

```diff
@@ -157,7 +157,13 @@ Template.create.events({
       last.previousSibling.focus();
     }
   },
+  'change .instancemodsinput': function(event, template) { // Function to be triggered on change event of the moderator email input
```

Comment:
@singhsanket143 putting comments at the end of the line is not the best practice, can you move it up or down somewhere? Thanks.

Reply:
@peter-hank Updated the comments position